packages/app/src/components/home/HomePage.tsx

```diff
@@ -60,6 +60,7 @@ const useCatalogStyles = makeStyles({
     '& img': {
       height: 80,
       width: 80,
+      "object-fit": 'contain',
```

Comment:
This should be `objectFit`, since it's JSS. `jss-camel-case` plugin doesn't complain in this particular version, however if Backstage ever updates to MUI v5, it will start raising warnings.

Reply:
Resolved in the latest commit.